ingestion/src/metadata/ingestion/source/pipeline/dbtcloud/client.py

```diff
@@ -67,37 +67,23 @@ def get_jobs(self) -> Optional[List[DBTJob]]:
         """
         list jobs for an account in dbt cloud
         """
-        try:
-            job_path = f"{self.config.jobId}/" if self.config.jobId else ""
-            result = self.client.get(
-                f"/accounts/{self.config.accountId}/jobs/{job_path}"
-            )
-            if result:
-                job_list = DBTJobList(**result).Jobs
-                return job_list
-        except Exception as exc:
-            logger.debug(traceback.format_exc())
-            logger.error(f"Unable to get job info :{exc}")
-
-        return None
+        job_path = f"{self.config.jobId}/" if self.config.jobId else ""
+        result = self.client.get(f"/accounts/{self.config.accountId}/jobs/{job_path}")
+        if result:
+            job_list = DBTJobList(**result).Jobs
+            return job_list
```

Comment:
we should not remove the exception handling here, instead we should prepare a new method for test connection

Reply:
prepared a new method for the test connection